src/Fmt_ast.ml

```diff
@@ -303,18 +303,41 @@ let rec fmt_longident (li: Longident.t) =
       cbox 0 (fmt_longident li1 $ wrap "(" ")" (fmt_longident li2))
 
 
-let fmt_constant const =
+let fmt_constant (conf: Conf.t) const =
+  let ocaml_whitespace = function
+    | '\n' | '\t' | '\r' | ' ' -> true
+    | _ -> false
+  in
+  let escape_char c =
+    let code = Char.to_int c in
+    match conf.Conf.escape_chars with
```

Comment:
Could you change this to use a type ascription `(c: Conf.t)` on the argument above to be uniform with the rest of the code.